connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedConnect.java

```diff
@@ -963,11 +962,19 @@ public Set<WorkerHandle> activeWorkers() {
         return workers().stream()
```

Comment:
I'm unable to comment on that line directly but the `ObjectMapper` initialized in the line above can be removed now.

Reply:
Ah, good catch! Done.